extensions/static-marks-bookmarks/src/index.tsx

```diff
@@ -19,68 +26,151 @@ export default function Command() {
 }
 
 function SearchListItem({ searchResult }: { searchResult: LinkResult }) {
+  const parents = [{ text: searchResult.parents.join(" > "), icon: Icon.Folder }];
+
   return (
     <List.Item
       title={searchResult.name}
+      subtitle={searchResult.description}
+      icon={getFavicon(searchResult.url)}
```

Comment:
I find that some square favicons can look a bit jarring. What do you think about applying a Mask?
```suggestion
      icon={getFavicon(searchResult.url, { mask: Image.Mask.Circle })}
```

Note: This is only supported in `@raycast/utils@1.4.11`

Reply:
Not so sure about this one, it adds consistency, but there are some beautiful favicons out there that look kind of odd after this treatment:

<img width="1000" alt="static-marks 2022-11-15 at 14 30 06" src="https://user-images.githubusercontent.com/140042/201931930-fee25edc-8efc-4b74-a84f-7587bcc198be.png">

Reply:
Wishing for an api that returns a favicon for a website with some padding and a white background....

Reply:
With `Image.Mask.RoundedRectangle` it looks a bit better IMO, what do you think @peduarte?

<img width="1000" alt="static-marks 2022-11-15 at 15 45 12" src="https://user-images.githubusercontent.com/140042/201948662-dd8136cb-ccec-4b67-878b-d08b67c1f8da.png">

Reply:
I went with `Image.Mask.RoundedRectangle` and added a more convenient fallback 😄 

![image](https://user-images.githubusercontent.com/140042/201953078-2410544e-895f-4788-b1c8-6cdb3dc15d3c.png)

Reply:
Looks good to me! Good call.